src/termdict/fst_termdict/streamer.rs

```diff
@@ -78,8 +78,8 @@ pub struct TermStreamer<'a, A = AlwaysMatch>
 where
     A: Automaton,
 {
-    fst_map: &'a TermDictionary,
-    stream: Stream<'a, A>,
+    pub fst_map: &'a TermDictionary,
+    pub stream: Stream<'a, A>,
```

Comment:
Adding `pub` was an easy way to access the inside of a `TermStreamer`.
@fulmicoton Do you know if it is possible to reduce the visibility to the `termdict` module ?

Reply:
`pub(crate)` would be a good start.